sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosException.java

```diff
@@ -44,7 +44,7 @@
 public class CosmosException extends AzureException {
     private static final long serialVersionUID = 1L;
 
-    private static final ObjectMapper mapper = Utils.getSimpleObjectMapper();
+    private static final ObjectMapper mapper = new ObjectMapper();
```

Comment:
BTW, request diagnostics also use an ObjectMapper. don't we need to do similiar fix to that?